docs/user/otp-deployment.md

```diff
@@ -0,0 +1,89 @@
+# Setting Up OTP Deployment Servers on Amazon Web Services (AWS)
+
+This document describes how to setup a load balancer server and an UI server for deployment using OTP DataTools, versions prior to Pull Request #506.
+
+Steps:
+
+1. Create the UI server (AWS S3 and CloudFront)
+2. Create the backend load balancer (AWS EC2).
+
+Assumptions for IBI-hosted deployments:
+
+1. You have access to an IBI Group Amazon Web Services (AWS) environment.
+2. Server instance types exist on AWS.
+3. A "key file name" exists.
+4. A Virtual Private Cloud (VPC) with two subnets exists on AWS.
+5. You are an admin for DataTools.
+
+## UI "Server": S3 Buckets and CloudFront
```

Comment:
This high level section about UI deserves a quick summary (similar to your summary below for `Backend server: Create a load balancer, and copy properties to DataTools`).